README.md

```diff
@@ -6,12 +6,65 @@
 
 ## Previous efforts referenced
 
+- [*A Benchmark of Ocular Disease Intelligent Recognition: One Shot for Multi-Disease Detection [1]* ](https://doi.org/10.1007/978-3-030-71058-3_11): This is the original paper associated with the ODIR-5k dataset, written by the authors of the dataset.
+  
+  -  It establishes baseline benchmarks for the dataset by using eight deep convolutional neural networks (CNN): Vgg-16, ResNet-18, ResNet-50, ResNeXt-50, SE-ResNet-50, SE-ResNeXt-50, Inception-v4, Densenet and CaffeNet. These CNN's were referenced in inspiring our choice of CNN.
+  
+  - In the authors model, left and right fundus images are fed into seperate models, and a feature fusion method (element-wise sum, element-wise multiplication or concatenation) is used to combine the output of both models. The authors emphasize that an occular disease recognition model should evaluate both the left and right fundus images to provide correct diagnosis. Rather than implementing a feature fusion technique, we decided to optionally fuse left and right fundus images as part of our pre-processing pipeline.  
+
+- [*Algorithms For Massive Data [2]*](https://github.com/GrzegorzMeller/AlgorithmsForMassiveData): Our model was initially forked from the Grzegorz Meller's model. His model was chosen as a starting point due to it's relatively simple implementation (which aided us in understanding the entire model).
+  
+  -  It is spread across a series of Jupyter-notebook's hosted on Google Colab, which are easy to quickly test and run.
```

Comment:
double check capitalization of jupyter-notebooks (also should be notebooks not notebook's)